app/controllers/application_controller.rb

```diff
@@ -137,4 +139,10 @@ def theme_chosen
     end
   end
 
+  protected
+
+  def configure_permitted_parameters
```

Comment:
hmm, as per the devise docs to permit additional attribute we need to permit then like this in application controller.